bin/git-browse-ci

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+set -e -o pipefail
```

Comment:
I get 128 exit code when no upstream is given after this modification...

Reply:
It seems to be the combination of `set -e` and `git rev-parse --abbrev-ref HEAD 2&> /dev/null` (specifically the `2&> /dev/null`). 

Changing it to the shorthand version `&> /dev/null` or the longhand version `>/dev/null 2>&1` fixes it for me.